src/Uno.Toolkit.Skia.WinUI/SkiaCompact.skia.cs

```diff
@@ -0,0 +1,314 @@
+﻿#nullable enable
+
+using System;
+using System.Runtime.CompilerServices;
+using SkiaSharp;
+using static SkiaSharp.SKImageFilter;
+
+namespace Uno.Toolkit.Skia.WinUI;
+
+internal static class SkiaCompat
+{
+	private static readonly bool _isSkiaSharp3OrLater = typeof(SKPaint).Assembly.GetName().Version?.Major >= 3;
```

Comment:
maybe could use something like:
```cs
private static readonly CoreImpl _impl = typeof(SKPaint).Assembly.GetName().Version?.Major >= 3
	? new V3UnsafeImpl()
	: new LegacyImpl();

internal interface CoreImpl
{
	SKImageFilter SKImageFilter_CreateBlur(float sigmaX, float sigmaY);
	...
}

internal static SKImageFilter SKImageFilter_CreateBlur(float sigmaX, float sigmaY) => _impl.SKImageFilter_CreateBlur(...);
```
and then you can have a `SkiaCompat.skia.legacy.cs` and a `SkiaCompat.skia.V3.cs`

Reply:
Yeah, for now I just stole this from https://github.com/unoplatform/uno/blob/master/src%2FUno.UI.Composition%2FSkiaCompat.skia.cs